sass/components/_post.scss

```diff
@@ -652,11 +652,8 @@
     }
     .loading-screen {
         position: relative;
-        height: 40px;
+        height: calc(40px + 1em);
```

Comment:
I needed to add 1em to not cause a jump when hiding the loader, but I didn't need to conditionally add it 🤷‍♂️